charts/akeyless-secure-remote-access/README.md

```diff
@@ -1,6 +1,7 @@
 # Akeyless Secure Remote Access
 
 Combines both Zero Trust Bastion and SSH-Bastion capabilities.
+In addition, There is [Zero Trust Portal](https://docs.akeyless.io/docs/access-resources-remotely#connect-from-the-secure-remote-access-portal) capability.
```

Comment:
I would add @EldadH89 for docs review and descriptions

Reply:
done